GameData/SpaceDust/Patches/SpaceDustResourcesCRP.cfg

```diff
@@ -1,6 +1,18 @@
 // This controls visibility of resources in theUI
 // -----------------------------------
 
+@SPACEDUSTSETTINGS:NEEDS[CommunityResourcePack]
+{
```

Comment:
This should not be CRP. should apply only to the mods that are using these resources, to prevent pointless mods shown in the UI